### DISCUSSION TITLE
$router.push on SSR

### DISCUSSION THREAD

**Discussion by toooni** [nuxt/nuxt#17601] [Questions]
Hi

Summary: `$router.push()` inside compoenent/page on SSR results in a 301 instead of 302 redirect. Can I change this?

I am currently checking in a component if an order still has a balance and am then redirecting with `router.push()` to the confirmation page. If this is happening in the client (browser) everything is fine. It also works if it's a full page load but results in a 301 instead of a 302 redirect. Is there an option I can pass to the router to use 302 (temporary since the state of the order may change) redirects?

**Comment by fiamma06** [comment]
Also interested in this question

**Reply by toooni** [reply]
I am now using navigateTo (https://v3.nuxtjs.org/guide/directory-structure/pages#programmatic-navigation).
